### ISSUE TITLE
can´t remove directories in v1.0

### ISSUE THREAD

**Issue by mg563** [tranxuanthang/lrcget#6]
Hi, 

first, I wanted to thank the developer for creating such a useful software, it has already found many missing lrc files in my library and was super simple to use. 

The new version it’s a big improvement, especially with the included player and lrc lyrics viewer, helps a lot to find if the lyrics do sync correctly. (Previously I was transferring the songs to my phone, and using poweramp to check this). Kudos!

As mentioned in the title I wanted to report that once a directory is added it can´t be removed. I already tried restarting the app, but does not allow to delete directories. I'm using the latest update in Win 10 x64. 

Also, as a couple of suggestions for future releases: 
-an option to only show track with or without lrc files.
-an option to search and overwrite lrc files (for those cases that the lrc was found somewhere else, like www.lyricsify.com, and lyrics are out of sync).  Not sure if it could display the new found lrc files, before overwriting the old file, to check it the new result is a better synched match or not. 
-would be nice if clicking on any part of the synchronized lyrics, would take you to the same time stamp of the playing track. This is supper useful to make sure lyrics are correctly synched. 
-On the track’s menu, it would be nice if a checkmark under the lyric’s column, could show which track also contains embedded lyrics or not. And for those that have embedded lyrics but no lrc, display the unsynched lyrics instead when playing the track.
-a dark theme, I like the pink design, but darker themes are easier on the eyes specially for those who spend lots of hours in front of a screen. Maybe a dark background with pink accents could be added. 

Thanks again for such an useful  tool!

**Comment by tranxuanthang** [comment]
Hey, I'm really glad you like my work!

I've just released v0.1.1, could you check if directories removal is working now?

https://github.com/tranxuanthang/lrcget/releases/tag/0.1.1

**Comment by tranxuanthang** [comment]
I'll close this to clean things up, but please feel free to reopen if you are still having this issue.

**Comment by mg563** [comment]
Thanks for the update, here are my findings with 0.1.2

-Directories can be removed now.
-The problem is that I ran into a different issue, I tried searching lrc for the song "Natalie Gang - Raging", and the program showed that an LRC result found, but nothing was actually downloaded. Another strange thing happened, after removing the directory for that song, and adding the folder back again (in order to rescan it), this time it did not show track, as if the directory was empty.
I uninstalled and tried searching for the lrc with version 0.5 and it showed that nothing was found. So I guess that the actual lrc is not available yet in the database, but v 0.1.2 is showing false positive results and this might cause some problems when rescanning that same folder.
